### ISSUE TITLE
Add how to contribute in Repo

### ISSUE THREAD

**Issue by Durgesh4993** [swapagarwal/swag-for-dev#1109]
- [ ] I've checked that this isn't a duplicate issue.

This helps to beigners to contributors, who intrested to contribute in projects

**Comment by aslafy-z** [comment]
We have https://github.com/swapagarwal/swag-for-dev/blob/master/CONTRIBUTING.md.
What do you think is missing in this document?

**Comment by Tejas-Sharma-dev** [comment]
Yes, Since we already have contributing.md file in git, which all all information about contribution. we don't want any dedicated page for it.

**Comment by aslafy-z** [comment]
Let's close this issue then
